### ISSUE TITLE
Fix drafter install instructions for Mac OSX

### ISSUE THREAD

**Issue by chris48s** [DemocracyClub/UK-Polling-Stations#1400]
See https://github.com/DemocracyClub/UK-Polling-Stations/pull/1391#issuecomment-398789958

**Comment by VirginiaDooley** [comment]
This issue could not be resolved because Drafter v5 cannot be used with current versions of Xcode. 

For now, we have to rely on non mac users to implement design system classes to the API documentation. In the mean time, we will explore alternative object builders that can render objects in cases when they exist and when they don't exist - as is the case with elections.

**Comment by GeoWill** [comment]
For reference on why we use api blueprint system: https://github.com/DemocracyClub/ec-api-proxy/issues/3#issuecomment-772834670

TLDR: we want multiple documented examples for a a given endpoint - e.g. what a `/postcode/...` response looks like when there is/isn't an address picker.
